polyfills/Element/prototype/placeholder/polyfill.js

```diff
@@ -1,4 +1,7 @@
 Object.defineProperty(Element.prototype, 'placeholder', {
+	get:function(){
+		return this.getAttribute('placeholder') ;},
```

Comment:
Line break before `}`, whitespace after `function()`.

We have linting rules in the root of the project.  Can you run them?